app/js/views/reminders/reminder-item.jsx

```diff
@@ -1,12 +1,19 @@
+/* global TwitterCldr, TwitterCldrDataBundle */
+
 import React from 'components/react';
 import moment from 'components/moment';
+import 'components/cldr/en';
+import 'components/cldr/core';
 
 const COLOURS = ['red', 'orange', 'green', 'blue', 'violet'];
 
 export default class ReminderItem extends React.Component {
   constructor(props) {
     super(props);
 
+    TwitterCldr.set_data(TwitterCldrDataBundle);
```

Comment:
As discussed offline, I filed an issue https://github.com/twitter/twitter-cldr-js/issues/78
I agree it's not ideal but it works.